### ISSUE TITLE
CPU pegged out running salt-master (after running command) 

### ISSUE THREAD

**Issue by waynew** [saltstack/salt#26301]
I'm running salt 2015.8 in an attempt to see if it resolves another issue. However, when I run any command that connects to the network, one of the salt-master processes runs away with my CPU. It looks to be rapidly polling `/dev/autofs`. A *lot*. Like 10s = 163,255 lines of strace output, with 156,458 of those containing 'poll'.

According to htop, it's been running for 6 minutes now, still at 100% CPU.

I can kill that particular process, which drops the CPU back down to a respectable level, though if I run any other commands, it pegs out again.

It appears that any command that I run produces the same behavior, and I can run several different commands with no other processes ramping up their CPU usage. 

```
Salt Version:
           Salt: 2015.8.0-260-g909556a
 
Dependency Versions:
         Jinja2: 2.8
       M2Crypto: 0.22
           Mako: 1.0.1
         PyYAML: 3.11
          PyZMQ: 14.7.0
         Python: 2.7.10 (default, May 26 2015, 04:16:29)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.2
          ioflo: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
       pycrypto: 2.6.1
        timelib: Not Installed
 
System Versions:
           dist:   
        machine: x86_64
        release: 4.0.6-1-ARCH
```

From lsof:
```
systemd       1              root   14u     unix 0xffff880019cd2680       0t0    3352476 /run/systemd/private type=STREAM
systemd       1              root   15r      CHR             10,235       0t0       1127 /dev/autofs
```

strace output:
```
10851 poll([{fd=14, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=15, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=14, events=POLLIN}, {fd=15, events=POLLIN}], 2, 0) = 0 (Timeout)
10851 poll([{fd=14, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=15, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=14, events=POLLIN}, {fd=15, events=POLLIN}], 2, 0) = 0 (Timeout)
10851 poll([{fd=14, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=15, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=14, events=POLLIN}, {fd=15, events=POLLIN}], 2, 0) = 0 (Timeout)
10851 poll([{fd=14, events=POLLIN}], 1, 0) = 0 (Timeout)                           
10851 poll([{fd=15, events=POLLIN}], 1, 0) = 0 (Timeout)                          
```
It goes on for a while

**Comment by aboe76** [comment]
I can confirm this behavior, on my archlinux box too.

**Comment by aboe76** [comment]
is this the same issue? https://github.com/saltstack/salt/issues/26366

**Comment by waynew** [comment]
It sure looks like it

**Comment by waynew** [comment]
Looks like there are actually other files with the same FD under lsof. I'm not currently on the bad branch, but this is what I have right now:

```
salt-mast   664  root   15u     IPv4              14661      0t0      TCP mymaster:4505->myminion:55606 (ESTABLISHED)
salt-mast   665  root   15u     unix 0xffff880175ec9880      0t0    13859 /var/run/salt/master/master_event_pull.ipc type=STREAM
salt-mast   671  root   15u  a_inode               0,11        0     6873 [eventfd]
salt-mast   672  root   15u  a_inode               0,11        0     6873 [eventfd]
salt-mast   675  root   15u  a_inode               0,11        0     6873 [eventfd]
salt-mast   678  root   15u  a_inode               0,11        0     6873 [eventfd]
salt-mast   680  root   15u  a_inode               0,11        0     6873 [eventfd]
salt-mast   684  root   15u  a_inode               0,11        0     6873 [eventfd]
```
(interesting point of fact: I don't currently have any salt commands running)

**Comment by cachedout** [comment]
Please try the fix in #26443

**Comment by waynew** [comment]
(I mentioned over there that I tried it and it works. Thanks!)
